storage/src/qmdb/current/ordered/mod.rs

```diff
@@ -0,0 +1,26 @@
+//! _Ordered_ variants of a [crate::qmdb::current] authenticated database. These variants maintain
+//! the lexicographic-next active key of each active key, allowing for exclusion proofs.
+
+use crate::qmdb::{
+    any::{ordered::fixed::Update, FixedValue},
+    current::proof::OperationProof,
+};
+use commonware_cryptography::Digest;
+use commonware_utils::Array;
+
+pub mod fixed;
+
+/// Proof information for verifying a key is not currently active in the database.
+#[derive(Clone, Eq, PartialEq, Debug)]
+pub enum ExclusionProof<K: Array, V: FixedValue, D: Digest, const N: usize> {
+    /// For the KeyValue variant, we're proving that a span over the keyspace exists in the
+    /// database, allowing one to prove any key falling within that span (but not at the beginning)
+    /// is excluded.
```

Comment:
```suggestion
    /// Proves that a span over the keyspace exists in the database, allowing any key
    /// falling strictly within that span (but not at the beginning) to be proven excluded. 
    /// The span's start key cannot be excluded since it is active.
```